### PR TITLE
Remaster of the Panel API part.

### PR DESCRIPTION
**Pull request by Astarsys**
- Added descriptions
- Added examples
- Added meta descriptions
- Removed / from call titles
- Added parameters to tables

### CODE REVIEW

File: docs/panel-api/resources/user.md [hunk 1]
```diff
@@ -1,106 +1,169 @@
...
-description: /user
+title: User
+description: API calls on work with users in the admin panel.
 ---
 
+# User
+
+API calls on work with users in the admin panel.
+
+API path: `/user`.
```
Comment (fenik17) [inline docs/panel-api/resources/user.md:10]: `/panel/user`

**Review by fenik17 — CHANGES_REQUESTED** [review]
Big job. Respect.

**Comment by Astarsys** [comment]
Thank you

File: docs/panel-api/resources/dealer/activation_code.md [hunk 2]
```diff
@@ -1,89 +1,187 @@
...
 
-    {
-        "success": true,
-        "count": <int> // count of actually updated codes
-    }
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/panel/dealer/activation_code/create' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "fa7bf873fab9333144e171372a321b06", "count": 10, "tariff_id": 12457, "bonus_amount": 3, "free_days": 5}'
+
```
Comment (fenik17) [inline docs/panel-api/resources/dealer/activation_code.md:60]: Missed ```

File: docs/panel-api/resources/dealer/index.md [hunk 3]
```diff
@@ -1,66 +1,128 @@
...
+
+#### examples
+
+=== "cURL"
+
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/panel/dealer/get_info' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "fa7bf873fab9333144e171372a321b06"}'
+
```
Comment (fenik17) [inline docs/panel-api/resources/dealer/index.md:30]: Missed ```

File: docs/panel-api/resources/dealer/password.md [hunk 4]
```diff
@@ -1,29 +1,45 @@
...
+| new_password | New password for the dealer, 6 to 20 printable characters. | string |
+
+#### example
+
+=== "cURL"
+
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/panel/dealer/password/update' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "fa7bf873fab9333144e171372a321b06", "old_password": "qwerty", "new_password": "B1r7d@Y"}'
```
Comment (fenik17) [inline docs/panel-api/resources/dealer/password.md:33]: Missed ```

File: docs/panel-api/resources/subpaas/index.md [hunk 5]
```diff
@@ -1,70 +1,113 @@
...
-* 13 – If the dealer
-    * is not paas
-    * has different status than NOT_BLOCKED
-    * his tariff doesn't allow subpaases
+=== "cURL"
 
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/panel/subpaas/create' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "fa7bf873fab9333144e171372a321b06", "password": "B1r7d@Y", "title": "Company", "email": "email@company.com", "jur_name": "Company", "jur_country": "Finland", "link_monitoring": "company.com"}'
```
Comment (fenik17) [inline docs/panel-api/resources/subpaas/index.md:68]: Missed ```

File: docs/panel-api/resources/subpaas/index.md [hunk 6]
```diff
@@ -1,70 +1,113 @@
...
-    * is not paas
-    * has different status than NOT_BLOCKED
-    * his tariff doesn't allow subpaases
+=== "cURL"
+
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/panel/subpaas/list' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "fa7bf873fab9333144e171372a321b06"}'
+
```
Comment (fenik17) [inline docs/panel-api/resources/subpaas/index.md:105]: Missed ```

File: docs/panel-api/resources/subpaas/index.md [hunk 7]
```diff
@@ -88,20 +131,39 @@ Get list of all subpaases for dealer. Dealer id will be taken from the session.
...
-    * is not paas
-    * has different status than NOT_BLOCKED
-    * his tariff doesn't allow subpaases
+=== "cURL"
+
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/panel/subpaas/read' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "fa7bf873fab9333144e171372a321b06", "subpaas_id": 97834}'
+
```
Comment (fenik17) [inline docs/panel-api/resources/subpaas/index.md:161]: Missed ```

File: docs/panel-api/resources/subpaas/password.md [hunk 8]
```diff
@@ -1,29 +1,45 @@
...
-    * The dealer is not paas
-    * The dealer has different status than NOT_BLOCKED
-    * The dealer's tariff doesn't allow subpaases
-    * Found subpaas is in DELETED status
+=== "cURL"
+
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/panel/subpaas/password/change' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "fa7bf873fab9333144e171372a321b06", "subpaas_id": "99874", "new_password": "Fr1d@Y$"}'
```
Comment (fenik17) [inline docs/panel-api/resources/subpaas/password.md:29]: Missed ```

File: docs/panel-api/resources/subpaas/session.md [hunk 9]
```diff
@@ -1,25 +1,39 @@
...
-#### errors
+#### examples
+
+=== "cURL"
+
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/panel/subpaas/session/create' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "fa7bf873fab9333144e171372a321b06", "subpaas_id": 97834}'
+
```
Comment (fenik17) [inline docs/panel-api/resources/subpaas/session.md:30]: Missed ```

**Review by fenik17 — CHANGES_REQUESTED** [review]

**Review by fenik17 — APPROVED** [review]

**Pull request merged**